src/CodeGen_Hexagon.cpp

```diff
@@ -2324,11 +2183,9 @@ bool CodeGen_Hexagon::possiblyCodeGenWideningMultiply(const Mul *op) {
     Ops.clear();
     //   Vdd.uh=vmpy(Vu.ub,Rt.ub)
     //   Vdd.uw=vmpy(Vu.uh,Rt.uh)
-    Patterns.push_back(Pattern(WPICK(wild_u16x128, wild_u16x64) *
-                               WPICK(wild_u16x128, wild_u16x64),
+    Patterns.push_back(Pattern(wild_u16x2W * wild_u16x2W,
```

Comment:
I just noticed this, are these supposed to be double vector widths? I'd expect these instructions to take single vectors and return a double vector.